staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode.go

```diff
@@ -22,6 +22,28 @@ import (
 	"github.com/fxamacker/cbor/v2"
 )
 
+var simpleValues *cbor.SimpleValueRegistry = func() *cbor.SimpleValueRegistry {
+	var opts []func(*cbor.SimpleValueRegistry) error
+	for sv := 0; sv <= 255; sv++ {
+		// Reject simple values 0-19, 23, and 32-255. The simple values 24-31 are reserved
+		// and considered ill-formed by the CBOR specification. We only accept false (20),
+		// true (21), and null (22).
+		switch sv {
+		case 20: // false
+		case 21: // true
+		case 22: // null
+		case 24, 25, 26, 27, 28, 29, 30, 31: // reserved
```

Comment:
The comments seem to suggest we're only accepting false, true, nil, but doesn't this case also allow the reserved values?

Reply:
The reserved values are always rejected. The library doesn't make it possible to accept them since they're considered malformed (https://github.com/fxamacker/cbor/blob/02b69dbb52f4ecf450b3aa5e9a04b7a0b4bf409a/valid.go#L315-L317).